### PR TITLE
feat: add LEPTON_WORKSPACE_TOKEN to secret if flag --include-workspace-token is specified

### PR DESCRIPTION
**Pull request by bddppq**

### CODE REVIEW

File: leptonai/cli/photon.py [hunk 1]
```diff
@@ -598,7 +602,8 @@ def run(
 
         if include_workspace_token:
             console.print("Including the workspace token for the photon execution.")
-            _add_workspace_token_secret_var_if_not_existing(conn)
+            secret = list(secret)
+            _add_workspace_token_secret_var_if_not_existing(conn, secret)
```
Comment (Yangqing) [inline leptonai/cli/photon.py:606]: Directly add

if "LEPTON_WORKSPACE_TOKEN" not in secret:
        secret.append("LEPTON_WORKSPACE_TOKEN")

here?

File: leptonai/cli/photon.py [hunk 2]
```diff
@@ -66,7 +66,9 @@ def _get_most_recent_photon_id_or_none(conn: Connection, name: str) -> Optional[
     return photon_ids[0] if photon_ids else None
 
 
-def _add_workspace_token_secret_var_if_not_existing(conn: Connection):
+def _add_workspace_token_secret_var_if_not_existing(
```
Comment (Yangqing) [inline leptonai/cli/photon.py:69]: for clarity we can call it "create_workspace_token_secret_var_if_not_existing"

File: leptonai/cli/photon.py [hunk 3]
```diff
@@ -84,6 +86,8 @@ def _add_workspace_token_secret_var_if_not_existing(conn: Connection):
             "Failed to create secret LEPTON_WORKSPACE_TOKEN.",
             exit_if_4xx=True,
         )
+    if "LEPTON_WORKSPACE_TOKEN" not in secret:
+        secret.append("LEPTON_WORKSPACE_TOKEN")
```
[the remark this answers is not in the document]
Comment (bddppq) [reply leptonai/cli/photon.py]: done

File: leptonai/cli/photon.py (hunk 2 above)
Comment (bddppq) [reply leptonai/cli/photon.py]: done

**Pull request merged**